src/lib/var.c

```diff
@@ -438,7 +437,11 @@ static void fetch_zval_from_symbol_table(
 						xdebug_var_maybe_destroy_ht(myht, is_temp);
 						goto cleanup;
 					}
+#if PHP_VERSION_ID >= 70400
```

Comment:
Ah, I see why this didn't work. There is another maybe_destroy on line 437, which also needs to be #ifdef'd

Reply:
Done as discussed